src/main/java/io/fusionauth/client/FusionAuthClient.java

```diff
@@ -311,6 +311,22 @@ public FusionAuthClient setTenantId(UUID tenantId) {
         .go();
   }
 
+  /**
+   * Creates an authenticatio API key. You can optionally specify an Id for the application, if not provided one will be generated.
+   *
+   * @param authenticationKeyId (Optional) The Id of the authentication key. If not provided a secure random UUID will be generated.
+   * @param request The request object that contains all of the information used to create the APIKey.
+   * @return The ClientResponse object.
```

Comment:
The Id is not a UUID, it is a string. The fix for this will be in the client lib.